nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ScanAttribute.java

```diff
@@ -97,13 +107,23 @@
             .addValidator(StandardValidators.createRegexValidator(0, 1, false))
             .defaultValue(null)
             .build();
+    public static final PropertyDescriptor DICTIONARY_ENTRY_METADATA_DEMARCATOR = new PropertyDescriptor.Builder()
+            .name("Dictionary Entry Metadata Demarcator")
```

Comment:
Yes please. That recommendation was put in place after a lot of the code already existed, so what we are trying to enforce is that any new properties follow that model and people will go back and update existing properties as necessary. There have been many discussions about it, but one serious issue is that if a property name is changed (to make it more readable, rename it, fix a typo, etc.) the value stored in the `flow.xml.gz` file will not be picked up when the flow is read by NiFi and that data can be lost. So the combination of static `name` and malleable `displayName` solves this issue.